Makefile

```diff
@@ -25,25 +25,34 @@ clean:
 
 publish: clean
 	mkdir -p build
-#	npm run publish 2> /dev/null
+	npm run publish 2> /dev/null
```

Comment:
publish to /dev/null ?

Reply:
it suppresses build output and was there before from API-M